wordpress-mobile/WordPress-iOS
# Aztec: Implements Calypso PreProcessor

### Details:
In this PR we're implementing a Pre Processor that converts a Calypso-Generated Document into (hopefully) Valid HTML, which is (again, hopefully) Aztec Friendly.

Fixes #7446

### To test:
1. Make sure the `CalypsoProcessorTests` are green
2. Create a Post in Calypso Web, with multiple newlines + a pre element
3. Edit the Post created in (2) in Aztec
4. Make sure the world is a happy place again. With actual newlines and Paragraph Breaks.

Needs review: @diegoreymendez 
Thanks in advance Diego!
